samples/gltf_viewer.cpp

```diff
@@ -96,9 +96,9 @@ struct App {
     } scene;
 
     struct ColorGradingOptions {
-        bool enabled = false;
+        bool enabled = true;
         int quality = static_cast<int>(ColorGrading::QualityLevel::MEDIUM);
-        int toneMapping = static_cast<int>(ColorGrading::ToneMapping::ACES_LEGACY);
+        int toneMapping = static_cast<int>(ColorGrading::ToneMapping::ACES);
```

Comment:
I am not sure that's what we want to do here. This tone mapper is darker which will create bigger mismatches with other renderers and may make comparisons more difficult.

Reply:
Non-legacy looks better but yeah good point -- I changed it back.

Reply:
It shouldn't: the only difference between legacy and non-legacy is exposure. Legacy is a slightly brighter version of non-legacy. You can get the same result by just changing your camera exposure and/or your light intensities.

Reply:
The exposure difference is 1/0.6.

Reply:
Ah interesting -- understood!  I think it looks pretty good right now actually, I think I'll go ahead and merge this.